tests/integration_tests_blueprint/README.md

```diff
@@ -0,0 +1,20 @@
+# Docl integration tests blueprint:
+The following docl blueprint provides a solution for executing the Cloudify integration tests on an Openstack/AWS environment.
+
+
+# Usage:
+Execution of the blueprint is done using the Cloudify CLI and requires pre-configuring the blueprint inputs file beforehand.
+* Install the Cloudify CLI by running  `pip install cloudify==3.4.0` in your virtual environment.
+* To have the tests run using your custom branch/organization, edit `docl-base-types.yaml` file accordingly.
+* Enter your cloud provider credentials under `os-inputs.yaml`/`aws-inputs.yaml` depending on the cloud provider of your choice.
+* Execute `cfy local install -p <path to blueprint> -i <path to blueprint inputs>`.
+
+
+# Custom operations:
+In addition to providing a way to execute the integration tests on a remote machine, the blueprint also provides some custom CI related operations that would be executed on the remote machine.
+## Upload docl image to S3:
+Once rebuilt, the Cloudify manager docl image can be compressed and uploaded to S3. This image can later be consumed by docl as an alternative for rebuilding the image from scratch.
+
+To execute the operation state the S3 credentials as well as destination bucket to the `config/upload_op_inputs.json`. After editing the inputs file, execute the operation using:
+
+`cfy local execute -w execute_operation -p <path to upload_op_inputs.json>`
```

Comment:
aws-inputs.yaml :
ec2_region_name default should be 'eu-central-1'
remove duplicate image_id
change instance_type to m3.large
change rebuild default to false
change run_integration_tests - out default false